backend/benefit/setup.cfg

```diff
@@ -7,6 +7,7 @@ ignore = E309
 exclude = migrations,snapshots
 max-line-length = 120
 max-complexity = 10
+ignore = E203, W503
```

Comment:
Having a comment explaining why this is done could be clarifying like e.g. in https://github.com/City-of-Helsinki/yjdh/blob/develop/backend/shared/setup.cfg#L8-L11
```python
# E203 and W503 are disabled and W504 enabled for compatibility with Black.
# For details see "Why are Flake8's E203 and W503 violated?" in Black FAQ at
# https://black.readthedocs.io/en/stable/faq.html
```